la_palma_overview/__init__.py

```diff
@@ -66,6 +66,19 @@ def clock2img(rows, cols):
         transform=ax.transAxes,
     )
 
+    try:
+        run_id = sfc.main_page().run_id
+        if run_id is not None:
+            ax.text(
+                0.5, 0.0, 'Run {0: 3d}'.format(run_id),
+                horizontalalignment='center',
+                verticalalignment='center',
+                fontsize=60, color='red',
+                transform=ax.transAxes,
+            )
+    except:
+        log.exception('exception in clock2img:')
```

Comment:
Technically this text is not needed as logging items know in which function they were thrown and this can be added to the log output. So a message like `Could not get `run_id` would be more helpful.